internal/support/notifications/router.go

```diff
@@ -122,9 +120,13 @@ func metricsHandler(w http.ResponseWriter, r *http.Request) {
 	t.LiveObjects = t.Mallocs - t.Frees
 
 	w.Header().Set("Content-Type", "application/json")
-	encode(t, w)
-
-	LoggingClient.Debug(fmt.Sprintf("Fetched these metrics for the notifications service: {%v}:", t))
+	enc := json.NewEncoder(w)
+	err := enc.Encode(t)
+	// Problems encoding
```

Comment:
Why not use encode(t, w) as before? the encode() function does error handling so you don't need it below.

See internal/support/notifications/utils.go line 28

Reply:
Will do (I had noticed that some routers were using _encode()_, while others weren't, so I wanted to make that consistent across the routers). But I agree that it's cleaner to use _encode()_ instead: I'll make it so 👍

Reply:
Never mind, I was looking at the definition of configHandler(), in reference to the note above. Making the use of configHandler() consistent in metricsHandler() as well!